pkg/metrics/metrics.go

```diff
@@ -981,6 +986,14 @@ func NewLegacyMetrics() *LegacyMetrics {
 			Help:       "Number of services events labeled by action type",
 		}, []string{LabelAction}),
 
+		ServiceProgrammingDuration: metric.NewHistogramVec(metric.HistogramOpts{
+			ConfigName: Namespace + "_service_programming_duration_seconds",
```

Comment:
There's an existing metric `policy_implementation_delay` which is somewhat similar to this, in the interest of having consistent metric naming, I'm wondering if we make this something like `service_implementation_delay`?

Reply:
I've renamed the metric to `service_implementation_delay`